core/templates/components/main-nav/main-nav.twig

```diff
@@ -0,0 +1,388 @@
+
+  {% if has_search == true %}
+<nav {{ attributes }} id="su-main-nav" class="su-main-nav su-main-nav--mobile-search {{ modifier_class }}" aria-label="Main">
+  {% else %}
+<nav {{ attributes }} id="su-main-nav" class="su-main-nav {{ modifier_class }}" aria-label="Main">
+  {% endif %}
+  <button id="su-main-nav__toggle" class="su-main-nav__toggle {{ button_modifier_class }}" aria-expanded="false">Menu</button>
+  <ul class="su-main-nav__menu-lv1">
+    <li class="su-main-nav__item-lv1">
```

Comment:
My guess is that these items are in here for preview? Have you given thought to how you would like to add them? Would it work best to add the markup for the link items as a region (variable) or loop through an array/object of data?

Reply:
Ohh..I see what you mean. I guess we covered this in the meeting - this is fairly new to me so I'm glad JB was there to answer your question.